api/envoy/type/tracing.proto

```diff
@@ -0,0 +1,111 @@
+syntax = "proto3";
+
+package envoy.type;
```

Comment:
I think we need to fix this so that it is in a versioned package, starting with v2, which will then be copied to v3. @htuch can you advise on the right thing to do here?